### PR TITLE
Minor improvements to "We Do Not Need Senior Developers"

### PR DESCRIPTION
**Pull request by andrewmy**
Fixed some typos, wording, and fixed "codebase" vs "code base" consistency.

### CODE REVIEW

File: source/_posts/2020/2020-03-02-we-do-not-need-senior-developers-we-need-senior-code-bases.md [hunk 1]
```diff
@@ -74,62 +74,62 @@ Now let's pause. **What misconceptions have you spotted?** Here are 3:
 - 3 seniors developers have more experience than 1 senior developer
 - you can learn only from people
 
-There all 3 are `false`.
+All 3 of those are `false`.
```
Comment (TomasVotruba) [inline source/_posts/2020/2020-03-02-we-do-not-need-senior-developers-we-need-senior-code-bases.md]:
```suggestion
All 4 of those are `false`.
```

One more suggestion from 
twitter

File: source/_posts/2020/2020-03-02-we-do-not-need-senior-developers-we-need-senior-code-bases.md [hunk 2]
```diff
@@ -74,62 +74,62 @@ Now let's pause. **What misconceptions have you spotted?** Here are 3:
...
+Often, senior developers are hired to orientate in a terrible codebase, they can't apply their experience with clean code, and they can't build a robust application with their high standards. They have to dig through the legacy code and try to fix leaking pipes.
 
 ## Senior Codebase is the Best Teacher
 
 <blockquote class="blockquote text-center">
     Monkey see, monkey doo.
 </blockquote>
 
-That's how our brain works; we're just like monkeys. We go to a new project, and we instinctively copy patterns we see. If there is [broken window all over the code](https://blog.codinghorror.com/the-broken-window-theory/), we make more broken windows.
+That's how our brain works; we're just like monkeys. We go to a new project, and we instinctively copy patterns we see. If there is a [broken window all over the code](https://blog.codinghorror.com/the-broken-window-theory/), we make more broken windows.
```
Comment (TomasVotruba) [inline source/_posts/2020/2020-03-02-we-do-not-need-senior-developers-we-need-senior-code-bases.md]: Probably better "broken windows all over", as it's many

File: source/_posts/2020/2020-03-02-we-do-not-need-senior-developers-we-need-senior-code-bases.md (hunk 1 above)
Comment (andrewmy) [reply source/_posts/2020/2020-03-02-we-do-not-need-senior-developers-we-need-senior-code-bases.md]: Right, I can read but not count 😂

**Comment by TomasVotruba** [comment]
Thanks

**Pull request merged**